src/auto/functions.rs

```diff
@@ -786,6 +786,13 @@ pub fn main_depth() -> i32 {
 //    unsafe { TODO: call ffi::g_markup_collect_attributes() }
 //}
 
+pub fn markup_escape_text(text: &str) -> Option<String> {
```

Comment:
This can't return NULL AFAIU

Maybe some of the other markup related functions can also be enabled now?

Reply:
I presume that following should get rid of Option:
```
[object.function.return]
nullable = false
```
In this case this doesn't work correctly right now.

Reply:
It works correctly for me
```toml
    [[object.function]]
    name = "markup_escape_text"
    [object.function.return]
    nullable = false
```

Reply:
@EPashkin: Please don't break indent! :scream:

Reply:
Oh yes, I will fix comment